### PR TITLE
[CP] LRSD-7596 - Customer can view details page with customerPublishingDate > now if navigating directly to it

### PR DESCRIPTION
**Pull request by abfranca**
https://liferay.atlassian.net/browse/LRSD-7596

### CODE REVIEW

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java [hunk 1]
```diff
@@ -410,6 +420,45 @@ private boolean _hasEarlyPublishAccess(Jwt jwt) {
 		return false;
 	}
 
+	private boolean _hasIssuePermission(Jwt jwt, JSONObject issueJSONObject) {
+		JSONObject fieldsJSONObject = issueJSONObject.getJSONObject("fields");
+
+		String publishingStatus = fieldsJSONObject.optString(
+			"publishingStatus");
+
+		LocalDateTime publishingDate = _parsePublishingDate(
```
Comment (kylembischof) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java:429]: can we move this `publishingDate` initialization inside the `publishingStatus.equals("Ready for Publishing")` if statement. that way if its not ready for publishing, none of this date logic has to execute

Comment (abfranca) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java]: Done!

**Review by kylembischof — APPROVED** [review]

**Comment by ryanschuhler** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/156862

**Comment by ryanschuhler** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/156867

**Pull request closed without merging**